outputs.tf

```diff
@@ -23,33 +23,3 @@ output "bigquery_tables" {
   value       = google_bigquery_table.main
   description = "Map of bigquery table resources being provisioned."
 }
-
-output "dataset_project" {
```

Comment:
Is this raising an error? It looks like a valid attribute to me.

Reply:
Is there any benefit from this output?

One could just do:

module.bq.bigquery_dataset.project instead

(versus module.bq.dataset_project).